networks/acala/project.yaml

```diff
@@ -56,6 +56,11 @@ dataSources:
           filter:
             module: balances
             method: Transfer
+        - handler: handleTokenTransfer
```

Comment:
That changes to fix that problem:
https://github.com/nova-wallet/subquery-nova/pull/136/files#r912531444
?

Reply:
Right, the order is what we set in `project.yaml`

`currencies.Transferred` event should be the last `transfer` event